helm/charts/oathkeeper/values.yaml

```
@@ -286,6 +286,13 @@ deployment:
   #      lines, adjust them as necessary, and remove the curly braces after 'annotations:'.
   #      e.g.  sidecar.istio.io/rewriteAppHTTPProbers: "true"
 
+  # -- Configure horizontal pod autoscaler for deployment
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
```

```suggestion
    maxReplicas: 5
```

Just so someone won't say that we recommend 100 replicas 😉